src/blocks/ui/components/variants.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { compactMetric, formatNumber } from '../../../utils';
+
+import Lift from './lift';
+
+const { __ } = wp.i18n;
+
+const VariantsList = styled.div`
+	padding-bottom: 30px;
+
+	& h2 {
+		color: #4767df;
+	}
+
+	.altis-analytics-block-variant {
+		margin: 30px 0 40px;
+
+		&--fallback {
+			margin-bottom: 20px;
+			border-bottom: 1px solid rgba( 0, 0, 0, .2 );
+			padding-bottom: 20px;
+		}
+	}
+
+	ul {
+		list-style: none;
+		display: flex;
+		flex-wrap: wrap;
+		margin: 10px 0 0;
+		padding: 0;
+	}
+
+	li {
```

Comment:
Still feel like an alternating light/dark background color for rows would make it more readable, but not a blocker to get this in and worth waiting until we get feedback about the end result to make design tweaks.

Reply:
I think with the big stripe further up that it would look odd using a background colour to alternate rows, would be a bit too much noise in the UI. Let's see what Noel thinks and come back to it.

Reply:
That's fair. 👍
I'm not in love with the big blue stripe, either, tbh, but `¯\_(ツ)_/¯`